packages/did/src/IdenityRegistry.js

```diff
@@ -1,7 +1,7 @@
 const tic = require('tic.api.js');
 const mamClient = require('mam.tools.js');
 
-const { encodeToDid, decodeFromDid } = require('./mnid');
+const { encodeTangleId, decodeTangleId } = require('./mnid');
```

Comment:
Avoid using `TangleId`. Instead, always use `TangleID`. Be aware of the sequence.